static/app/views/settings/projectMetrics/projectMetrics.tsx

```diff
@@ -67,7 +67,9 @@ function ProjectMetrics({project}: Props) {
         )}
       </TextBlock>
 
-      <PermissionAlert project={project} />
+      {hasMetricsExtrapolationFeature(organization) ? null : (
```

Comment:
shouldn't this be `hasCustomMetricsExtractionRules`?

Reply:
actually we want to check if `hasExtractionRules` is available. If `True` we do not display the permission warning. I have update the code. Thanks